misc/tox.ini

```diff
@@ -66,3 +66,8 @@ changedir = {toxinidir}/style/
 deps =
 commands =
   python run-uncrustify.py --modify
+
+[testenv:history]
+changedir = {toxinidir}/history/
+commands =
+  python3 run-all-checks.py
```

Comment:
If I just run python run-all-checks.py, then it uses python2.7 instead of python3. Does someone know how to fix this?

Reply:
You can set `basepython = python3` (see above), but I thought the environment would inherit this setting.

Reply:
after setting it in the environment, it worked as desired.